docs/source/contributing.rst

```diff
@@ -0,0 +1,30 @@
+============
+Contributing
+============
+
+Contributors to the collection should take their time to read these guidelines,
+in order to avoid certain common pitfalls and needless frustration during code review.
+
+These guidelines may change, depending on evolution of the project.
+However, all changes should be perfomed with consideration for other developers
+workflow and for projects using edpm-ansible collection.
+
+General recommendation
+++++++++++++++++++++++
+
+Test your code locally, if it is at all possible, to shorten the development loop
+and minimize unnecessary use of CI. Before commiting your changes, run `pre-commit`_
```

Comment:
Could have sense to explain how to run a full pre-commit scan? (e.g. `pre-commit run --all-files`)